.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -7,6 +7,18 @@ assignees: ''
 
 ---
 
+**IF YOU DON'T REMOVE AND UNDERSTAND THE NEXT FEW LINES, THEN YOUR REPORT WILL BE CLOSED**
```

Comment:
This is way too harsh. I think we should rephrase it like this:

`PLEASE MAKE SURE YOU READ AND UNDERSTAND THE SOCIAL MEDIA SUPPORT CHANNELS. QUESTIONS ABOUT FLASHING, CONFIGURING, PILOTING MAY BE CLOSED WITHOUT FURTHER INTERACTION.`